operators/src/processing/interpolation/mod.rs

```diff
@@ -224,7 +225,7 @@ where
             ctx,
             sub_query,
         )
-        .filter_and_fill())
+        .filter_and_fill(CacheExpiration::NoCache)) // Filler tiles may be produced by gaps in the input data and we cannot be sure if the gaps are permanent or not. Thus, we do not cache them.
```

Comment:
Don't filler tiles have a useful cache time?

Reply:
c.f. my comment in the trello card.

Reply:
I think you are right that operators should not output gaps.
We could add a parameter to the filler that would input the expiration time. Then, at least each operator can decide on that.

Reply:
Please think of the implications. This will make all results where a tile is filled non-cachable. I bet that almost all queries will contain some filled tiles. There are operators (GdalSource, all based on SubQueryAdapter) that just don't output empty tiles and  rely on the filler to produce them. We probably need to pass something like a strategy to the filler to decide what the TTL of a tile created by it should be.

Reply:
I implemented a filler strategy to derive the cache hint from the surrounding tiles 9b5283436f1ac6f6344a5396a0d0ddda4be63040